<ui version="4.0" >
 <class>Kleo::NewCertificateUi::ListWidget</class>
 <widget class="QWidget" name="Kleo::NewCertificateUi::ListWidget" >
  <layout class="QVBoxLayout" name="verticalLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QListView" name="listView" >
     <property name="selectionBehavior" >
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="horizontalScrollMode" >
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="resizeMode" >
      <enum>QListView::Adjust</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QToolButton" name="addTB" >
       <property name="text" >
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="removeTB" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="upTB" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="downTB" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>addTB</sender>
   <signal>clicked()</signal>
   <receiver>Kleo::NewCertificateUi::ListWidget</receiver>
   <slot>slotAdd()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>13</x>
     <y>206</y>
    </hint>
    <hint type="destinationlabel" >
     <x>33</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>removeTB</sender>
   <signal>clicked()</signal>
   <receiver>Kleo::NewCertificateUi::ListWidget</receiver>
   <slot>slotRemove()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>43</x>
     <y>210</y>
    </hint>
    <hint type="destinationlabel" >
     <x>125</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>upTB</sender>
   <signal>clicked()</signal>
   <receiver>Kleo::NewCertificateUi::ListWidget</receiver>
   <slot>slotUp()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>203</x>
     <y>214</y>
    </hint>
    <hint type="destinationlabel" >
     <x>257</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>downTB</sender>
   <signal>clicked()</signal>
   <receiver>Kleo::NewCertificateUi::ListWidget</receiver>
   <slot>slotDown()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>245</x>
     <y>220</y>
    </hint>
    <hint type="destinationlabel" >
     <x>257</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotAdd()</slot>
  <slot>slotRemove()</slot>
  <slot>slotUp()</slot>
  <slot>slotDown()</slot>
 </slots>
</ui>
